test/python/pulse/test_builder.py

```diff
@@ -462,46 +466,6 @@ def test_snapshot(self):
 
         self.assertEqual(schedule, reference)
 
-    def test_call_schedule(self):
```

Comment:
I believe tests be kept until fully deprecated?

Reply:
not deprecated. this is moved to new unittest class which is a collection of tests involving the call instruction.